pkgpanda/__init__.py

```diff
@@ -545,7 +545,7 @@ def add_user(self, username, group):
             '-c', 'DCOS System User',
         ]
 
-        if group is not None:
+        if group is not None and group != username:
```

Comment:
Can we add a code comment for why the logic is not executed if `group == username`?